### DISCUSSION TITLE
OLED theme

### DISCUSSION THREAD

**Discussion by Zegnat** [jocmp/capyreader#422] [Feature Requests]
Currently the dark theme does not use pure black. Which is great for almost all use-cases, it results in contrasts that are easier on the eyes, fits with the Material style, etc.

But in the case of reading text on an OLED display, it would be great to get pure black backgrounds which will only have text actually light up.

An open-source example that shows this in option is [Feeder](https://news.nononsenseapps.com/) ([GitHub](https://github.com/spacecowboy/feeder), [F-Droid](https://f-droid.org/en/packages/com.nononsenseapps.feeder/)). When its theme is set to “Night”, it will apply a user specified “preferred dark theme”, which can be set to black for pure black backgrounds.

**Comment by jocmp** [comment]
Hey @Zegnat - I'm trying to take a simple approach theming to avoid too many options, but I'm a big fan of these pure black settings. I've used it in the past on Pocket Casts and loved it on Windows Phone back in the day.

I can't promise a timeline, but I'll see what I can do after I work on #332.

**Reply by Zegnat** [reply]
Awesome! No expectations of a timeline at all. Just noticed it had not been brought up yet so wanted to leave a nudge in its direction.

Thanks for all your work on Capy Reader. I am loving it!

**Reply by jocmp** [reply]
@Zegnat I meant to follow up here. The first pass at this feature is now available in version 2024.11.1077 under the "Display" settings.
